packages/dd-trace/src/profiling/profilers/wall.js

```diff
@@ -20,27 +59,73 @@ class NativeWallProfiler {
       process._stopProfilerIdleNotifier = () => {}
     }
 
-    this._record()
+    this._pprof.time.start({
+      intervalMicros: this._samplingIntervalMicros,
+      durationMillis: this._flushIntervalMillis,
+      sourceMapper: this._mapper,
+      customLabels: this._codeHotspotsEnabled,
+      lineNumbers: false })
```

Comment:
```suggestion
      lineNumbers: false
    })
```

Reply:
We're probably missing an eslint config for it. https://eslint.org/docs/latest/rules/object-curly-newline appears to be what we're looking for?